mm2src/coins/sia/spend_policy.rs

```diff
@@ -0,0 +1,340 @@
+// use super::address::v1_standard_address_from_pubkey;
+use crate::sia::address::Address;
+use crate::sia::blake2b_internal::{public_key_leaf, sigs_required_leaf, standard_unlock_hash, timelock_leaf,
+                                   Accumulator, ED25519_IDENTIFIER};
+use crate::sia::encoding::Encoder;
+use ed25519_dalek::PublicKey;
+use rpc::v1::types::H256;
+
+#[cfg(test)] use std::str::FromStr;
+
+const POLICY_VERSION: u8 = 1u8;
+pub trait Policy {}
```

Comment:
Why is this trait needed?

Reply:
good catch, artifact of a previous code pattern